src/core/metallicroughnessmaterial.cpp

```diff
@@ -493,6 +493,11 @@ WrappedSignal<OutputType> wrapParameterSignal(MetallicRoughnessMaterial *self, S
 
 MetallicRoughnessMaterial::MetallicRoughnessMaterial(Qt3DCore::QNode *parent)
     : QMaterial(parent)
+    , m_baseColorUsesTexCoord2(new QParameter(QStringLiteral("baseColorUsesTexCoord2"), bool(false)))
```

Comment:
shouldn't we go for baseColorUsesTexCoord1 instead since on the shaderSize the 2nd set of texture coords is vertexTexCoord1 ?

Reply:
you are right, ill change it